src/frontend/next/src/components/Posts/LoadAutoScroll.tsx

```diff
@@ -0,0 +1,58 @@
+import { useEffect, useRef } from 'react';
+
+import { Container, Button, Grid, createStyles, Theme } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+
+type Scroll = {
+  onScroll: Function;
+};
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    content: {
+      '& > *': {
+        padding: theme.spacing(2),
+        bottom: theme.spacing(4),
+      },
+    },
+  })
+);
+
+function LoadAutoScroll({ onScroll }: Scroll) {
+  const classes = useStyles();
+  const $buttonRef = useRef(null);
```

Comment:
See https://github.com/typescript-cheatsheets/react#forwardrefcreateref

Reply:
Not sure, if I fixed this correctly.